sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/README.md

```diff
@@ -293,6 +293,35 @@ The Azure Monitor Distro is a distribution package that facilitates users in sen
 
 Refer to [`Program.cs`](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/tests/Azure.Monitor.OpenTelemetry.AspNetCore.Demo/Program.cs) for a complete demo.
 
+### Log Scopes
+
+Log scopes allow you to add additional properties to the logs generated by your application.
+Although the Azure Monitor Exporter does support scopes, this feature is off by default in OpenTelemetry.
+To leverage log scopes, you must explicitly enable them.
+
+To include the scope with your logs, set `OpenTelemetryLoggerOptions.IncludeScopes` to `true` in your application's configuration:
+```csharp
+builder.Services.Configure<OpenTelemetryLoggerOptions>(options =>
+{
+    options.IncludeScopes = true;
+});
+```
+
+Once enabled, you can define a scope using a dictionary or any enumerable of KeyValuePair<string, object>.
+All logs written within the context of the scope will include the specified information.
+Azure Monitor will add these scope values to the Log's CustomProperties.
```

Comment:
please clarify the collision behavior for collision within diff. scopes (as they can be nested), and between scopes and log attributes.

Reply:
updated.